rktl_game/CMakeLists.txt

```diff
@@ -0,0 +1,206 @@
+cmake_minimum_required(VERSION 3.0.2)
+project(rktl_game)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  rospy
+  std_msgs
+)
+
+## System dependencies are found with CMake's conventions
```

Comment:
I prefer deleting all the stuff that we aren't going to use to make this more readable.

Reply:
done